lib/sarif/gosec_sarif.rb

```diff
@@ -0,0 +1,81 @@
+module Sarif
+  class GosecSarif < BaseSarif
+    GOSEC_URI = 'https://github.com/securego/gosec'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = GOSEC_URI
+      begin
+        json_obj = JSON.parse(scan_report.log(''))
+        issues = json_obj['Issues']
+        errors = json_obj['Golang errors']
+        parsed_errors = []
+        errors.each do |key|
+          key[1].each do |location|
+            location['uri'] = key[0]
+            parsed_errors << parse_error(location)
+          end
+        end
+        @logs = issues.concat parsed_errors
+      rescue JSON::ParserError
+        @logs = []
```

Comment:
In this case, I think you should output an error.  Maybe have `to_sarif` raise an exception?

Reply:
Its not an error that should lead to failure. If a ParseError is raised, it could mean that there was nothing logged by gosec for that repo because it either generated an error or it passed successfully. If theres an error, its handled in the build_invocations method